runtime/src/accounts_db.rs

```diff
@@ -3108,31 +3108,33 @@ impl AccountsDb {
     {
         let mut stored_accounts: HashMap<Pubkey, FoundStoredAccount> = HashMap::new();
         let mut original_bytes = 0;
-        let mut num_stores = 0;
-        for store in stores {
-            original_bytes += store.total_bytes();
-            let store_id = store.append_vec_id();
-            AppendVecAccountsIter::new(&store.accounts).for_each(|account| {
-                let new_entry = FoundStoredAccount { account, store_id };
-                match stored_accounts.entry(new_entry.account.meta.pubkey) {
-                    Entry::Occupied(mut occupied_entry) => {
-                        if new_entry.account.meta.write_version
-                            > occupied_entry.get().account.meta.write_version
-                        {
-                            occupied_entry.insert(new_entry);
+        let store_ids = stores
+            .into_iter()
+            .map(|store| {
+                original_bytes += store.total_bytes();
+                let store_id = store.append_vec_id();
```

Comment:
Will store_id be duplicated? if so, can we use a Hashset for ids?

Reply:
It will not be duplicated. But, there should only be 1.